Automattic/jetpack
# Jetpack plugin: Activate license key dropdown selection in Jetpack plugin's activation page.

Fixes #22913 

This is a follow up PR to #27609. It appears that SelectionScreen component is used in two separate page (`my-jetpack#/add-license` and `jetpack#/license/activation`). The original [PR](https://github.com/Automattic/jetpack/pull/27609) had already activated the license key dropdown selection in the **My Jetpack** package. This PR activates the one in the **Jetpack** plugin.  

#### Changes proposed in this Pull Request:
* Activate license key dropdown selection in Jetpack plugin's activation page.

#### Other information:


#### Jetpack product discussion
Context: 1202858161751496-as-1201858908716176

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Goto https://cloud.jetpack.com/pricing and purchase a product or two. (Make sure you do not activate the license key)
2. Launch a test site that uses a build of the Jetpack plugin based on this branch (Make sure you build the Jetpack plugin and My Jetpack package)
3. Connect your site to Jetpack, if it's not already
4. Go to Jetpack dashboard, click the **Activate your product license** now button.
<img width="2550" alt="Screen Shot 2022-12-16 at 3 26 05 PM" src="https://user-images.githubusercontent.com/56598660/208045713-62eb8032-0cd7-4705-bef7-a207988a2e7c.png">
5. Confirm that the license key dropdown selection appears in the activation page `jetpack#/license/activation` and you can activate the license key without problem.
<img width="1104" alt="Screen Shot 2022-12-16 at 3 28 12 PM" src="https://user-images.githubusercontent.com/56598660/208045971-7a4bc084-33ba-432c-b1ac-ff6ff8bb83c2.png">
6. Additionally make sure that the My Jetpack activation page `my-jetpack#/add-license` also works correctly.
